libraries/src/HTML/Helpers/Select.php

```diff
@@ -662,7 +670,7 @@ public static function radiolist(
                 $extra .= ((string) $k === (string) $selected ? ' checked="checked" ' : '');
             }
 
-            $html .= '<input type="radio" class="form-check-input" name="' . $name . '" id="' . $id . '" value="' . $k . '" '
+            $html .= '<input type="radio" class="form-check-input' . $class .'" name="' . $name . '" id="' . $id . '" value="' . $k . '" '
```

Comment:
```suggestion
            $html .= '<input type="radio" class="form-check-input' . $class . '" name="' . $name . '" id="' . $id . '" value="' . $k . '" '
```
PHPCS - this should be the last one.